docs/sdk-and-tools/indices/tokens.md

```diff
@@ -0,0 +1,35 @@
+---
+id: es-index-tokens
+title: Elasticsearch index: tokens
+---
+
+The `_id` field of this index is represented by token identifier of an ESDT token.
+
+| Field         | Description                                                                                                                 |
+|---------------|-----------------------------------------------------------------------------------------------------------------------------|
+| name          | The name field holds the name of the token. It contains alphanumeric characters only.                                       |
+| ticker        | The ticker field represents the unique combination of letters assigned to a token.                                          |
+| token         | The token represents the token identifier which has the form `ABCD-012345`.                                                 |
+| issuer        | The issuer field holds the address in a bech32 format of the issuer of the token.                                           |
```

Comment:
The issuer field holds the bech32 encoded address of the token's issuer.

Reply:
changed